pulp_rpm/app/viewsets.py

```diff
@@ -313,6 +691,17 @@ class CopyViewSet(viewsets.ViewSet):
 
     serializer_class = CopySerializer
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["create"],
+                "principal": ["authenticated"],
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.copy_rpmrepository",
```

Comment:
I am not sure this correct. CopyViewset involves operation on 2 repos: source and destination, so user need to have view permissions on both of them.
Since this is not an  action on the RepositoryViewset it has the following endpoint `pulp/api/v3/rpm/copy/`
I believe it makes sense to have a separate permission dedicated to the copy operation, however this is not enough.

Reply:
So we need helper functions to excert permissions on both repository parameters, correct?

Reply:
We do. so it is added.